dataprofiler/data_readers/data_utils.py

```diff
@@ -334,7 +344,7 @@ def reservoir(file: TextIOWrapper, sample_nrows: int) -> list:
         except StopIteration:
             break
         # Append new, replace old with dummy, and keep track of order
-        remove_index = rng.randrange(sample_nrows)
+        remove_index = rng.integers(0, int(sample_nrows), 1)[0]
```

Comment:
is the `int(...)` necessary, input above suggests type is an `int`

Reply:
fixed!